packages/react-native/Libraries/Types/CodegenTypes.d.ts

```diff
@@ -0,0 +1,36 @@
+/**
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @format
+ */
+
+import {EventSubscription} from '../vendor/emitter/EventEmitter';
+import {NativeSyntheticEvent} from './CoreEventTypes';
+// Event types
+// We're not using the PaperName, it is only used to codegen view config settings
+export interface BubblingEventHandler<T, PaperName extends string | undefined> {
+  (event: NativeSyntheticEvent<T>): void | Promise<void>;
+}
+
+export interface DirectEventHandler<T, PaperName extends string | undefined> {
+  (event: NativeSyntheticEvent<T>): void | Promise<void>;
+}
+// Prop types
+export type Double = number;
+export type Float = number;
+export type Int32 = number;
+export type UnsafeObject = any; // Object is forbidden in strict mode
+export type UnsafeMixed = any;
```

Comment:
Isn't this the same thing as here? https://github.com/facebook/react-native/blob/afb40e9f2a644647dda8a4b75478069c512d57cd/packages/react-native/types/modules/Codegen.d.ts#L42